src/Page/Auth.elm

```diff
@@ -526,7 +526,10 @@ viewRegisterForm ({ user } as model) =
                                     )
                                 ]
                                 []
-                            , text "Je m'engage à ne pas exploiter les données pour une utilisation commerciale."
+                            , text "Je m’engage à respecter les conditions d’utilisation (cf. "
+                            , a [ href Env.cguUrl ]
+                                [ text "conditions d'utilisation" ]
+                            , text ")"
```

Comment:
We should probably use `Markdown.simple` here.

Reply:
done